.gitignore

```diff
@@ -51,3 +51,6 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# pycharm projects files
+.idea
```

Comment:
could you remove this useless lines. (useless for OCA community, maybe not for you...)